tests/testthat/_snaps/target_sda.md

```diff
@@ -3,38 +3,38 @@
     Code
       out
     Output
-      # A tibble: 68 x 4
-         sector  year emission_factor_metric emission_factor_value
-         <chr>  <dbl> <chr>                                  <dbl>
-       1 cement  2020 projected                              1    
-       2 cement  2021 projected                              2    
-       3 cement  2022 projected                              3    
-       4 cement  2020 corporate_economy                      1    
-       5 cement  2021 corporate_economy                      2    
-       6 cement  2022 corporate_economy                      3    
-       7 cement  2020 target_b2ds                            1    
-       8 cement  2021 target_b2ds                            0.978
-       9 cement  2022 target_b2ds                            0.956
-      10 cement  2023 target_b2ds                            0.933
+      # A tibble: 68 x 6
```

Comment:
nit: are snapshots the most appropriate way of testing data frames? seems like formatting could easily cause false positive failures

Reply:
Not particularly, snapshots are a pretty brittle test, but they are left in as `target_sda` still has somewhat poor test coverage.